RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -9,6 +9,9 @@
 # run only if there are high pT jets
 jetsForCoreTracking = cms.EDFilter('CandPtrSelector', src = cms.InputTag('ak4CaloJetsForTrk'), cut = cms.string('pt > 100 && abs(eta) < 2.5'))
 
+jetsForCoreTrackingBarrel = cms.EDFilter('CandPtrSelector', src = cms.InputTag('ak4CaloJetsForTrk'), cut = cms.string('pt > 100 && abs(eta) < 2.5'), filter = cms.bool(False))
+jetsForCoreTrackingEndcap = cms.EDFilter('CandPtrSelector', src = cms.InputTag('ak4CaloJetsForTrk'), cut = cms.string('pt > 100 && abs(eta) > 1.4 && abs(eta) < 2.5'), filter = cms.bool(False))
```

Comment:
BTW, shouldn't `jetsForCoreTrackingBarrel` have `abs(eta) < 1.4` or something close, consistent with where `jetsForCoreTrackingEndcap` starts?

Reply:
thanks for the comment, I'm going to implement them

about the IB it was `CMSSW_12_0_X_2021-04-23-1100` (I'm moving to the `CMSSW_12_0_X_2021-05-04-1100` right now)

about the `filter = cms.bool(False)`, it is just to make sure no events are filtered out, and I'm going to add it to the `jetsForCoreTracking` module

about the eta acceptance, I do not want to force any restriction the geometry and training will make the job, while I explicitly applied to the endcap region the selection on eta in order to avoid to double counting the seeds (adding mainly fakes ....). I was also thinking to add a cluster removal in order to clean even further the environment, but I'm not sure it is the way to go